config/environments/development.rb

```diff
@@ -59,4 +59,7 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+  if Rails.env.development?
+    BetterErrors::Middleware.allow_ip! "0.0.0.0/0"
+  end
```

Comment:
[nits]
development.rbファイルだし、これでいいかも？(ちゃんと調べてない)
```suggestion
  BetterErrors::Middleware.allow_ip! "0.0.0.0/0"
```